ocs_ci/ocs/const_must_gather.py

```diff
@@ -0,0 +1,162 @@
+must_gather_files = {
```

Comment:
we already use one const. file, not sure another one needed

Reply:
@AviadP 
@clacroix12 offered to open a new folder for must_gather.